Directory.Packages.props

```diff
@@ -4,7 +4,7 @@
     <PackageVersion Include="Microsoft.NET.Sdk.Functions" Version="4.1.3" />
     <PackageVersion Include="Microsoft.Azure.Functions.Worker.Extensions.Abstractions" Version="1.1.0" />
     <PackageVersion Include="Microsoft.ApplicationInsights" Version="2.17.0" />
-    <PackageVersion Include="Microsoft.Azure.WebJobs" Version="3.0.32" />
+    <PackageVersion Include="Microsoft.Azure.WebJobs" Version="3.0.36-11991" />
```

Comment:
3.0.36 is now available on NUget - https://www.nuget.org/packages/Microsoft.Azure.WebJobs - please update